firebase-perf/src/main/java/com/google/firebase/perf/transport/RateLimiter.java

```diff
@@ -78,11 +86,16 @@ static float getSamplingBucketId() {
       final long capacity,
       final Clock clock,
       float samplingBucketId,
+      float fragmentBucketId,
       ConfigResolver configResolver) {
     Utils.checkArgument(
-        0.0f <= samplingBucketId && samplingBucketId < 1.0f,
+        0.0f <= samplingBucketId
```

Comment:
Recommend separating these checks. Makes it easy to debug with separate log statements (One for trace bucket ID and one for fragment).

Reply:
Done